toplevel/ccompile.ml

```diff
@@ -95,7 +95,7 @@ let create_empty_file filename =
 let check_pending_proofs filename =
   let pfs = Vernacstate.Declare.get_all_proof_names () [@ocaml.warning "-3"] in
   if not (CList.is_empty pfs) then
-    fatal_error (str "There are pending proofs: "
+    fatal_error (str "There are pending proofs in file " ++ str filename ++ str": "
```

Comment:
Maybe we should take a location?

Reply:
I thought of using locations but then either we say the error is at line 1 or we have to do something complicated to get the last line (which probably isn't much better in the end)

Reply:
We could use the location of the last command seen, but indeed, not sure about it.

I have a patch in serapi / jscoq that actually maps list of vernacs to something with structure, in this case we know when sections / proofs / etc... are open, so we could try to track the proof that is actually to blame, but indeed doing that right seems like some work ahead.